### PR TITLE
Updating Teams emergency calling and call routing policy PS cmdlet files

### PR DESCRIPTION
**Pull request by JensTrier**
Fixing some examples, adding more information and changing author

### CODE REVIEW

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

**Review by get-itips — APPROVED** [review]

**Review by tseward — APPROVED** [review]

File: skype/skype-ps/skype/New-CsTeamsEmergencyNumber.md [hunk 1]
```diff
@@ -17,46 +17,32 @@ schema: 2.0.0
...
-New-CsTeamsEmergencyNumber -EmergencyDialString <String> [-EmergencyDialMask <String>]
- [-OnlinePSTNUsage <String>] [-CarrierProfile <String>] [<CommonParameters>]
+New-CsTeamsEmergencyNumber [-EmergencyDialString <String>] [-EmergencyDialMask <String>]
+ [-OnlinePSTNUsage <String>] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
- This cmdlet supports to create multiple Teams emergency numbers.
+ This cmdlet supports to create multiple Teams emergency numbers. Used with TeamsEmergencyCallRoutingPolicy and 
+ only relevant for Direct Routing
```
Comment (dariomws) [inline skype/skype-ps/skype/New-CsTeamsEmergencyNumber.md:26]:
```suggestion
 This cmdlet supports creating multiple Teams emergency numbers. Used with TeamsEmergencyCallRoutingPolicy and only relevant for Direct Routing.
```

File: skype/skype-ps/skype/Remove-CsTeamsEmergencyCallingPolicy.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ Remove-CsTeamsEmergencyCallingPolicy [-Tenant <System.Guid>] [-Identity] <XdsIde
 PS C:>  Remove-CsTeamsEmergencyCallingPolicy -Identity testECP
 ```
 
- This example removes an existing Teams Emergency Calling policy with identity testECRP
+ This example removes an existing Teams Emergency Calling policy with identity testECP
```
Comment (dariomws) [inline skype/skype-ps/skype/Remove-CsTeamsEmergencyCallingPolicy.md:34]:
```suggestion
 This example removes an existing Teams Emergency Calling policy with identity testECP.
```

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**